charts/lfx-v2-project-service/templates/ruleset.yaml

```
@@ -30,9 +30,12 @@ spec:
         routes:
           - path: /projects
       execute:
-        #- authenticator: authelia
+        {{- if .Values.authelia.enabled }}
+        - authenticator: authelia
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 6 but found 8

(indentation)